afs/media/js/views/components/reports/observation.js

```diff
@@ -132,11 +159,6 @@ define([
                                 value: self.getRawNodeValue(self.resource(), 'observed'), 
```

Comment:
The "object observed during observation" appears twice here and also in the related object tab. I suspect that this should be deleted.